### PR TITLE
[Identify]: Add comma separated list of times support

### PR DESCRIPTION
**Pull request by loicgasser**
ping @oterral

### CODE REVIEW

**Comment by gjn** [comment]
What's the trigger for this?

**Comment by loicgasser** [comment]
Shop devs by @oterral

File: chsdi/views/features.py [hunk 1]
```diff
@@ -429,7 +429,11 @@ def _get_features_for_filters(params, layerBodIds, maxFeatures=None, where=None)
             # Filter by time instant
             if params.timeInstant is not None and hasattr(model, '__timeInstant__'):
                 timeInstantColumn = model.time_instant_column()
-                query = query.filter(timeInstantColumn == params.timeInstant)
+                timeInstantIndex = layerBodIds.index(layer)
+                timeInstant = params.timeInstant[timeInstantIndex]
```
Comment (gjn) [inline chsdi/views/features.py]: Don't we need a check that the index does not overflow here?

Comment (loicgasser) [inline chsdi/views/features.py]: yes you're right, not 100% safe. I would do that earilier during the validation process.

```
len(layers) == len(timestamps)
````
and if we have only one timestamp apply it to all the time enabled layers.

It's also an undocumented parameter, so we may be safe. thanks for your comment anyway, I'll adapt now.

**Comment by loicgasser** [comment]
@gjn I addressed your comment via https://github.com/geoadmin/mf-chsdi3/pull/2074/commits/b0e2e62f2c43919a4b7333f0bfff5b98f94a2487

File: chsdi/lib/validation/identify.py [hunk 2]
```diff
@@ -179,12 +179,29 @@ def tolerance(self, value):
...
+                            'Only years are supported as timeInstant parameter: provided value %s' % instant)
+                    if instant.isdigit():
+                        instant = int(instant)
+                    elif len(instant) == 0:
+                        instant = None
+                    else:
+                        raise HTTPBadRequest(
+                            'Please provide an integer for the parameter timeInstant: provided value %s' % instant)
+                    instants.append(instant)
+            elif nbTimeInstants == 1:
```
Comment (gjn) [inline chsdi/lib/validation/identify.py]: You miss the checks here in this code branch. (test taht the timeinstant is of correct lenght, is a digit, etc).

Comment (loicgasser) [inline chsdi/lib/validation/identify.py]: yes

**Comment by loicgasser** [comment]
done

File: chsdi/views/features.py (hunk 1 above)
Comment (gjn) [inline chsdi/views/features.py]: This is not done yet, is it?

Comment (loicgasser) [inline chsdi/views/features.py]: Done in validator

**Comment by gjn** [comment]
+1. Thanks alot.

**Pull request merged**